modules/ROOT/pages/backup-restore/restore-backup.adoc

```diff
@@ -131,7 +131,12 @@ For more information, see <<restore-cloud-storage>>.
 
 [NOTE]
 ====
-The `--temp-path` option is available starting from Neo4j 5.24.
+Neo4j 5.24 introduces the `--temp-path` option to address potential issues related to disk space when performing backup-related commands, especially when cloud storage is involved. 
+
+If `--temp-path` is not set, a temporary directory is created inside the current working directory for Neo4j.
```

Comment:
Hi! I had to double check: in this specific case, first it goes via the path defined in `--from-path` and then the current working directory